packages/devtools_app/lib/src/code_size/code_size_screen.dart

```diff
@@ -240,7 +241,19 @@ class _SnapshotViewState extends State<SnapshotView> with AutoDisposeMixin {
               axis: Axis.vertical,
               children: [
                 _buildTreemap(),
-                CodeSizeSnapshotTable(rootNode: snapshotRoot),
+                Row(
+                  children: [
+                    Flexible(
+                      child: CodeSizeSnapshotTable(rootNode: snapshotRoot),
+                    ),
+                    if (controller.callGraphRoot.value != null)
+                      Flexible(
```

Comment:
why flexible instead of expanded?
in general I think all uses of flexible should have a reason as the behavior is more surprising than expanded.

Reply:
because I want CodeSizeSnapshotTable and CallGraphWithDominators to take up equal space in the Row, so wrapping each with a Flexible gives each element an equal flex of 1